tools/packaging/packaging.psm1

```diff
@@ -234,14 +247,77 @@ function Start-PSPackage {
                 Force = $Force
             }
 
-            if($pscmdlet.ShouldProcess("Create $_ Package"))
-            {
+            if ($PSCmdlet.ShouldProcess("Create $_ Package")) {
                 New-UnixPackage @Arguments
             }
         }
     }
 }
 
+function New-TarballPackage {
+    [CmdletBinding(SupportsShouldProcess=$true)]
+    param (
+        [Parameter(Mandatory)]
+        [string] $PackageSourcePath,
+
+        # Must start with 'powershell' but may have any suffix
+        [Parameter(Mandatory)]
+        [ValidatePattern("^powershell")]
+        [string]$Name,
+
+        [Parameter(Mandatory)]
+        [string]$Version,
+
+        [switch] $Force
+    )
+
+    $packageName = "$Name-$Version-{0}-x64.tar.gz"
+    if ($Environment.IsWindows) {
+        throw "Must be on Linux or macOS to build 'tar.gz' packages!"
+    } elseif ($Environment.IsLinux) {
+        $packageName = $packageName -f "linux"
+    } elseif ($Environment.IsMacOS) {
+        $packageName = $packageName -f "osx"
+    }
+
+    $packagePath = Join-Path -Path $PWD -ChildPath $packageName
+    Write-Verbose "Create package $packageName"
+    Write-Verbose "Package destination path: $packagePath"
+
+    if (Test-Path -Path $packagePath) {
+        if ($Force -or $PSCmdlet.ShouldProcess("Overwrite existing package file")) {
+            Write-Verbose "Overwrite existing package file at $packagePath" -Verbose
```

Comment:
I guess we can remove `-Verbose`

Reply:
Here I put `-Verbose` explicitly to notify that the existing package is going to be removed.